apps/remix-ide/src/app.js

```diff
@@ -410,7 +410,7 @@ class AppComponent {
             if (params.call) {
               const callDetails = params.call.split('//')
               if (callDetails.length > 1) {
-                this.appManager.call('notification', 'toast', `initiating ${callDetails[0]} ...`)
+                this.appManager.call('notification', 'toast', `initiating ${callDetails[0]} and calling "${callDetails[1]}" ...`)
```

Comment:
so we limit it to two?

Reply:
no there can be multiple calls.

Reply:
here we are loging the name of the plugina dn the function call